Flutter/calendar/appointments.md

```diff
@@ -130,6 +130,86 @@ events.notifyListeners(CalendarDataSourceAction.reset, null);
 {% endhighlight %}
 {% endtabs %}
 
+## Notify listener
+
+By call this method whenever the datasource changes, to notify any clients the datasource may have. We have provided support to perform three operations like Add, Remove and Reset. NotifyListener is used to notify the dataSource may have changed.
```

Comment:
to notify any clients the datasource may have

Check and remove content related to clients. we need to notify the data source not clients

Reply:
I have checked and removed the content related to clients. I have updated the changes in my latest commit.